### PR TITLE
Frontend issue #161 about us page 

### PR DESCRIPTION
**Pull request by johndowd**
-moved components into about us folder & updated imports
-created about us page
-added missing image from boston.gov website

### CODE REVIEW

File: frontend/front/src/pages/Public/Pages/AboutUs/AboutUs.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+import mcec from "../../../../assets/images/MACleanEnergy.png"
+import cfb from "../../../../assets/images/CFB.png"
+import cfa from "../../../../assets/images/CFA.png"
+import heatsmart from "../../../../assets/images/HeatSmart.png"
+import pcb from "../../../../assets/images/powercorp-boston.jpeg"
+
+function AboutUs() {
+  return (
+    <Box>
+      <Grid item display="flex" flexDirection="column" alignItems="center" mr={4} ml={4}>
```
Comment (thiagobardini) [inline frontend/front/src/pages/Public/Pages/AboutUs/AboutUs.js:14]: I added the code below inside the GRID:

```
sx={{
color: 'var(--color-text-1) !important'
}}
```

File: frontend/front/src/pages/Public/Pages/AboutUs/AboutUs.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        <Typography
+          variant="h4"
+          mt={3}
+          mb={3}
+          color="#000000"
```
Comment (thiagobardini) [inline frontend/front/src/pages/Public/Pages/AboutUs/AboutUs.js:19]: Remove color

File: frontend/front/src/pages/Public/Pages/AboutUs/AboutUs.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        <Typography paragraph={true}>
+          The BHPA project is directed by ULEM Board Member Christopher Scranton, in coordination with core partners, listed below.
+        </Typography>
+        <img style={{ height: 100, 'max-width': '90%', mt: 3, mb: 3 }} src={ulem} />
+        <Typography
+          variant="h4"
+          mt={3}
+          mb={3}
+          textAlign="center"
+          color="#000000"
```
Comment (thiagobardini) [inline frontend/front/src/pages/Public/Pages/AboutUs/AboutUs.js:42]: Remove color

File: frontend/front/src/pages/Public/Pages/AboutUs/PartnerTile.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+import { Box, Grid, Typography, Divider } from "@mui/material";
+
+const PartnerTile = ({ partnerName, paragraphText, image, website }) => {
+
+  return (
+    <Grid
+      container
+      justifyContent="center"
+      alignItems="center"
+      color="#000000"
```
Comment (thiagobardini) [inline frontend/front/src/pages/Public/Pages/AboutUs/PartnerTile.js:11]: Remove color

File: frontend/front/src/pages/Public/Pages/AboutUs/AboutUs.js [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+        </Typography>
+        <Typography mb={3} paragraph={true}>
+          <b>The Urban League of Eastern Massachusetts (ULEM)</b> is the sponsor of the BHPA effort. Since 1919, ULEM has delivered workforce and economic development services and programs to increase self-reliance of residents of the Boston community and surrounding metropolitan areas. ULEM is a 501c3 nonprofit organization and one of the oldest affiliates within the National Urban League movement.
+        </Typography>
+        <Typography paragraph={true}>
+          The BHPA project is directed by ULEM Board Member Christopher Scranton, in coordination with core partners, listed below.
+        </Typography>
+        <img style={{ height: 100, 'max-width': '90%', mt: 3, mb: 3 }} src={ulem} />
```
Comment (thiagobardini) [inline frontend/front/src/pages/Public/Pages/AboutUs/AboutUs.js:36]: Remove the code below:
```
<img
  style={{ height: 100, "max-width": "90%", mt: 3, mb: 3 }}
  src={ulem}
/>
```
and replace it with this code using Box:
```

<Box
  component="img"
  src={ulem}
  sx={{
    height: 100,
    "max-width": "90%",
    mt: 3,
    mb: 3,
  }}
/>
```

File: frontend/front/src/pages/Public/Pages/AboutUs/PartnerTile.js [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+      marginTop={3}
+      marginBottom={3}
+    >
+      <Typography marginBottom={3} paragraph={true}>
```
Comment (thiagobardini) [inline frontend/front/src/pages/Public/Pages/AboutUs/PartnerTile.js:15]: Remove paragraph={true}

File: frontend/front/src/pages/Public/Pages/AboutUs/PartnerTile.js [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+      </Typography>
+      <a
+        href={website}
+        target='_blank'
+      >
+        <img
+          style={{ height: 100, marginBottom: 12 }}
+          alt={`logo of ${partnerName}`}
+          src={image}
+        />
```
Comment (thiagobardini) [inline frontend/front/src/pages/Public/Pages/AboutUs/PartnerTile.js:26]: Remove the code below:
```
       <img
          style={{ height: 100, marginBottom: 12 }}
          alt={`logo of ${partnerName}`}
          src={image}
        />
```
and replace it with this code using Box:
```

        <Box
          component="img"
          src={image}
          alt={`logo of ${partnerName}`}
          sx={{
            height: 100,
            mb: 3,
          }}
        />
```

File: frontend/front/src/pages/Public/Pages/AboutUs/PartnerTile.js [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+      justifyContent="center"
+      alignItems="center"
+      color="#000000"
+      marginTop={3}
+      marginBottom={3}
+    >
+      <Typography marginBottom={3} paragraph={true}>
+        <b>{partnerName}</b> {paragraphText}
+      </Typography>
+      <a
```
Comment (thiagobardini) [inline frontend/front/src/pages/Public/Pages/AboutUs/PartnerTile.js:18]: replace `<a href={website} target="_blank" rel="noopener noreferrer">`

File: frontend/front/src/pages/Public/Pages/AboutUs/PartnerTile.js [hunk 9]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Box, Grid, Typography, Divider } from "@mui/material";
```
Comment (thiagobardini) [inline frontend/front/src/pages/Public/Pages/AboutUs/PartnerTile.js:2]: Remove `Divider`

File: frontend/front/src/pages/Public/Pages/AboutUs/AboutUs.js [hunk 10]
```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import { Box, Grid, Typography, Divider } from "@mui/material";
```
Comment (thiagobardini) [inline frontend/front/src/pages/Public/Pages/AboutUs/AboutUs.js:2]: Remove `Divider`

**Review by thiagobardini — CHANGES_REQUESTED** [review]
Hi Jack, could you please make these changes before we merge it? Let me know if you have any questions. Thanks!

**Comment by johndowd** [comment]
@thiagobardini

**Review by thiagobardini — APPROVED** [review]
Hey John. Good job! Thanks!

**Pull request merged**